src/locales/es.js

```diff
@@ -18,10 +18,12 @@ export default {
   'login.form.email.invalid': 'Debe ingresar un email válido',
   'login.form.password.presence': 'Debe ingresar una contraseña para continuar',
 
+  // Menu
   'menu.about': 'ACERCA',
   'menu.contact': 'CONTACTO',
 
-  'signup.title': 'REGISTRARSE',
+  // SignUp
+  'signup.title': 'REGISRARSE',
```

Comment:
There is a missing 'T' here

Reply:
Yes, it had made fixed previously but in the merge, I didn't see